chia/util/initial-config.yaml

```diff
@@ -550,3 +546,18 @@ wallet:
 
   # Interval to resend unconfirmed transactions, even if previously accepted into Mempool
   tx_resend_timeout_secs: 1800
+
+simulator:
+
+  # Should the simulator farm a block whenever a transaction is in mempool
+  auto_farm: True
+
+  # The key used by the simulator to farm blocks and send transactions
+  key_fingerprint:
```

Comment:
Does this actually work, with no value?

Reply:
Yes it does, that's why I do the check for none in start_simulator